### DISCUSSION TITLE
When does pnpm create `_` type links vs direct links? And how can I control it? It's breaking our mono repo setup.

### DISCUSSION THREAD

**Discussion by sean256** [reduxjs/redux-toolkit#2179] [Q&A]
I'm an idiot, I posted in the wrong place. Carry on.

**Comment by phryneas** [comment]
This really does not seem like the place to find out why your bundler of choice bundles your project a certain way. How would we know?

**Reply by sean256** [reply]
@phryneas This isn't about my bundler, I want to know how to control the way `pnpm` creates links.

Just forget the bundler thing

Why is `pnpm` creating 2 different links to 2 different instances of the same module?

```
../../../../node_modules/.pnpm/@someScope+react@0.6.4_react-is@17.0.2/node_modules/@someScope/react
../../../../node_modules/.pnpm/@someScope+react@0.6.4_4db1f9ae395425ec4d56821dc57dd590/node_modules/@someScope/react
```

**Reply by phryneas** [reply]
Sorry, I meant "package manager". You are in the discussions for "Redux Toolkit". I have no idea why you are asking a pnpm question here.

**Reply by sean256** [reply]
OH lol

**Reply by sean256** [reply]
I'm just gonna shamefully walk away...
